Bottom: e952549640e99fa10908312efda036a7e057b440
Top:    a2d0a63518ccec362a19ea084a85f0538f977de2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-23 19:26:33 +0100

crosshair.c: Always allow snapping to the element mark (except in rat-draw mode)


---

diff --git a/src/crosshair.c b/src/crosshair.c
index 67613b6..da8bb92 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -862,7 +862,7 @@ FitCrosshairIntoGrid (LocationType X, LocationType Y)
   snap_data.y = y;
 
   ans = NO_TYPE;
-  if (!PCB->RatDraw && TEST_FLAG (SNAPPINFLAG, PCB))
+  if (!PCB->RatDraw)
     ans = SearchScreenGridSlop (Crosshair.X, Crosshair.Y,
                                 ELEMENT_TYPE, &ptr1, &ptr2, &ptr3);
